src/CFamily/CMake/Resources.resx

```diff
@@ -143,7 +143,7 @@
   </data>
   <data name="NoCompilationDatabaseFile" xml:space="preserve">
     <value>[CMake] Could not locate compilation database at '{0}'. Make sure that your project is configured correctly. 
-    See https://github.com/SonarSource/sonarlint-visualstudio/wiki for more information.</value>
+    See https://docs.sonarsource.com/sonarlint/visual-studio/getting-started/requirements/#language-specific-requirements for more information..</value>
```

Comment:
For this one there was a sublink -> https://docs.sonarsource.com/sonarlint/visual-studio/getting-started/requirements/#getting-started-vs2019-and-later however because it is version specific it felt wrong so I decided on using the parent link for this. LMK what you think would be better

Reply:
Yep, I agree that the parent link makes more sense.